lib/cards/__tests__/visual.spec.js

```diff
@@ -0,0 +1,17 @@
+import {
+  renderComponentForVisualTest,
+  takeSnapshot,
+  delay,
+} from '../../../jest.conf/visual.testUtils';
+
+describe.visual('Cards visual tests', () => {
+  const snapshotOptions = { widths: [1380], minHeight: 512 };
+  it('renders', async () => {
+    await renderComponentForVisualTest('CardsVisualTest');
+
+    // Wait for loading animation to complete
+    await delay(4000);
```

Comment:
Not oposed to this, but we could also disable this wait directly in the examples if the `VISUAL_TESTING` env variable is true? My only fear is that there could be a race condition since the percy test process runs independently of the docs where the visual test examples are displayed, If they both would be running in the same process I wouldnt mind since these callbacks would have been scheduled and executed in order, but being independent, I think there is a small posibility of a race condition, not sure.

On a diferent thought - I think we can also add a card loading example to the visual test where we dont check this env variable, and let the loader run for the snapshot.

Reply:
I'm not sure if I follow all nuance, but I think I get that it's fragile approach. Yes, we'd better improve this. 

How would this https://github.com/learningequality/kolibri-design-system/pull/1107/commits/f919dc49e7840a5aebafac24ce20439ada446ade look like?  I thought it'd be a bit more suitable pattern - to avoid having to make decision in example files themselves based on what environment we're running them.

I also added two visual tests for loading state.